terra-workspace-manager-performance/build.gradle

```diff
@@ -0,0 +1,83 @@
+buildscript {
+    repositories {
+        maven {
+            url "https://plugins.gradle.org/m2/"
+        }
+    }
+    dependencies {
+        classpath "gradle.plugin.com.google.cloud.tools:jib-gradle-plugin:2.4.0"
+    }
+}
+
+plugins {
+    id 'scala'
+    id 'java'
+    id 'com.github.lkishalmi.gatling' version '3.3.3'
+}
+
+def jibPluginId = 'com.google.cloud.tools.jib'
+final hasPlugin = project.getPlugins().hasPlugin(jibPluginId);
+if (hasPlugin) {
+    final Plugin plugin = project.getPlugins().getPlugin(jibPluginId)
+    println 'Plugin already applied - version ' + plugin.properties['jibPluginId']
+} else {
+    apply plugin: "com.google.cloud.tools.jib"
+}
```

Comment:
Why is this section needed? Does applying the jib plugin to this project conflict with the main project?

Reply:
Yes that's correct, the main project already reference the jib class path which causes a conflict in the performance subproject. I haven't found a better way to fix this yet.